modules/exploits/windows/http/git_lfs_rce.rb

```diff
@@ -0,0 +1,211 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
```

Comment:
This doesn't appear to be necessary. I don't see where any HTTP requests are being made and I'm still able to successfully use the module without it.